fetch/cosmwasm_contract/src/contract.rs

```diff
@@ -842,9 +903,19 @@ fn _verify_not_paused(env: &Env, paused_since_block: u64) -> HandleResult {
 }
 
 fn verify_swap_amount_limits(amount: Uint128, state: &State) -> HandleResult {
-    if amount < state.lower_swap_limit {
+    if amount < state.swap_min {
+        Err(StdError::generic_err(ERR_SWAP_LIMITS_VIOLATED))
+    } else if amount > state.swap_max {
+        Err(StdError::generic_err(ERR_SWAP_LIMITS_VIOLATED))
+    } else {
+        Ok(HandleResponse::default())
+    }
+}
+
+fn verify_reverse_swap_amount_limits(amount: Uint128, state: &State) -> HandleResult {
+    if amount < state.reverse_swap_min {
```

Comment:
Tbhis check should not be there - it is sorted in reverseSwap and refund calls by simply withdrawing from user maximum we can (either reverse swap fee iif possible, or whole amount if it smaller than reverse swap fee).

This check will make some swaps unprocessable, ultimatelly maing to whole bridge to halt (on top of that relayer service will keep restarting iunnecessarily) - this is especially critical when we get in to the situation (for whatever reason) when we would change limits and not adhere to strict procedure (pausing, flushing in-fligvht TXs, etc.).

There is huge difference between reverse_swap_max and min - max is absolutelly criticall to be checked here due to security reasons in order to limit & slown down rug pull should relayer key get compromised.
But reverse_swap_min is completelly unnecessary - as I was saying multiple times before - it can be completelly droped.